src/api/public/apidocs-new/paths/source_project_name_cmd_copy.yaml

```diff
@@ -0,0 +1,52 @@
+post:
+  summary: Copy the entire project.
+  description: |
+    The `copy` command copies the entire project from the source project provided by `oproject` to the target project provided by `project`.
+
+    The params `makeolder` and `makeoriginolder` update the relevant vrev.
+    The param `withbinaries` ensures we copy also the binaries.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: oproject
+      schema:
+        type: string
+      required: true
+      description: Origin project name
+      example: "home:Admin"
+    - in: query
+      name: opackage
+      schema:
+        type: string
+      required: true
+      description: Origin package name
+      example: "ctris"
+    - in: query
+      name: makeolder
+      schema:
+        type: string
+      description: Make target older, the source vrev is bumped by two numbers and target by one. This parameter is interpreted as `true` if present, `false` otherwise.
+      example: 1
+    - in: query
+      name: makeoriginolder
+      schema:
+        type: string
+      description: Make origin older, the source vrev is extended and target is guaranteed to be newer. This parameter is interpreted as `true` if present, `false` otherwise.
+      example: 1
+    - in: query
+      name: withbinaries
+      schema:
+        type: string
+      description: Copies also binaries on copy command. This parameter is interpreted as `true` if present, `false` otherwise.
+      example: 1
```

Comment:
According to [this line](https://github.com/openSUSE/open-build-service/blob/3743e2a80736cabfa2f714ba8e0c36ec48523ee4/src/api/app/controllers/source_controller.rb#L623) and [this line](https://github.com/openSUSE/open-build-service/blob/3743e2a80736cabfa2f714ba8e0c36ec48523ee4/src/api/app/controllers/source_controller.rb#L630) we only check for the existence of `makeolder`, `makeoriginolder` and `withbinaries` but we don't check their values.

Then having something like `makeolder=0` or `makeolder=bla` will be considered true. Only when the parameter is not passed won't apply it.

However, this is an inconsistency with the backend that is expecting a boolean value.
Testing with  `makeolder=bla`,  this is the error message we get: `copy failed: not boolean`.

We can declare this as a `string` for now and postpone some research with [this card](https://trello.com/c/i2q5t19Q/2446-research-about-parameters-acting-as-parameters).